README.md

```diff
@@ -10,14 +10,22 @@ For guides on installing, using and configuring CoffeeLint, head over
 To suggest a feature, report a bug, or general discussion, head over
 [here](http://github.com/clutchski/coffeelint/issues/).
 
-## Team
+## :rotating_light: Team :rotating_light:
+
+There are no future plans for CoffeeLint. Only major bugs will be addressed
```

Comment:
i think at this point people should just fork it if they care. none of us have used coffeescript in years. i think it's fine to stick a 🍴 in this one.